### PR TITLE
CI: Add version bump GitHub Actions workflow

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Description

Adding version bump GitHub Actions workflow.

### Motivation and Context

- Related: #11590
- [x] Make bumping commit
- [x] Create branch and PR

### Behaviors

- Change `Package.appxmanifest` using bash command 'sed' and wildcard.
- Create a new commit with `peter-evans/create-pull-request@v4`.
  - Create a branch like 'yair100/bump-up-to-3.0'
  - Create a commit on behalf of an author who runs this workflow
- Create a new PR with `peter-evans/create-pull-request@v4`.
  - Add a label `automation`
  - Add a reviewer who runs this workflow

### Validation

- [x] Built and ran the app
- [x] Tested the changes for accessibility

### Screenshots

They will be going to create something like this PR
image

### CODE REVIEW

File: .github/workflows/files_version_bump.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - name: Fetch resources
+              run: |
+                  sed -i "s/<Identity Name=\"\(.*\)\" Publisher=\"\(.*\)\" Version=\"\(.*\)\" \/>/<Identity Name=\"\1\" Publisher=\"\2\" Version=\"${{ inputs.files_new_version }}\" \/>/g" "src/Files.App (Package)/Package.appxmanifest"
+            - name: Commit changes
+              uses: EndBug/add-and-commit@v9
+              with:
+                  author_name: yaira2
```
Comment (yaira2) [inline .github/workflows/files_version_bump.yml:21]: Can we set the author to @filescommunity?

Comment (onein528) [reply .github/workflows/files_version_bump.yml:21]: I tried to use this 'public beta' `on-behalf-of` trailer, however it doesn't work.

**Comment by onein528** [comment]
@yaira2 Please tell me:

### 1. Create a Secret of Personal Access Token in https://github.com/settings/tokens/new (No expiration, repo checked)
  image

### 2. Add that in this repository https://github.com/files-community/Files/settings/secrets/actions
  image

**Comment by yaira2** [comment]
Should we create a GitHub account dedicated for running these workflows?

**Comment by onein528** [comment]
I think this is enough. but yeah probably should create a bot only for this organization something like `files-community[bot]`.

**Pull request closed without merging**